internal/reports/REPORT.md

```diff
@@ -0,0 +1,23 @@
+## Harbourbridge Report
+A Harbourbridge report consists of the following elements:
+
+#### Summary
+Defines the overall quality of the conversion perfomed by Harbourbridge.
+
+#### Ignored statements
+Defines the statements in the source schema which have been ignored by Harbourbridge. For example, View related statements are currently ignored by Harbourbridge.
+#### Conversion duration
```

Comment:
Conversion metadata - making it inline with actual object naming

Reply:
Done